rpi5.yaml

```diff
@@ -206,3 +212,32 @@ parameters:
           MACHINE: "raspberrypi5"
           SOC_FAMILY: "bcm2712"
           ZEPHYR_DOM0_DIR: "zephyr"
+
+  DOMD_ROOT:
+    desc: "Domd root device"
+    mmc:
+      default: true
+      overrides:
+        components:
+          domd:
+            builder:
+              conf:
+                - [DOMD_OVERLAYS, "%{SOC_FAMILY}-%{MACHINE}-mmc.dtbo"]
+                - [MACHINE_FEATURES:append, " domd_mmc"]
+        images:
+          full:
+            partitions:
+              rootfs:
+                gpt_type: B921B045-1DF0-41C3-AF44-4C6F280D3FAE # Linux aarch64 root
+                type: raw_image
+                size: 8 GiB
+                image_path: "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
+
+    usb:
+      overrides:
+        components:
+          domd:
+            builder:
+              conf:
+                - [DOMD_OVERLAYS, "%{SOC_FAMILY}-%{MACHINE}-usb.dtbo"]
```

Comment:
if you add here below it will fix bootargs:
`                - [DOMD_BOOTARGS, "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\\\/dev\\\/sda2 rootfstype=ext4 rootwait"]
`

Reply:
it could be not sda2 of course - use what is right